_posts/2020-05-01-native-command-mode-tools.adoc

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: 'Ideas on Quarkus command-mode, UI-based apps and releasing via Github Actions'
+date: 2020-05-10
+tags: development-tips
+synopsis: Ideas on how to use Quarkus command-mode to develop native applications with web frontends; built with Github Actions.
+author: dplatz
+---
+:imagesdir: /assets/images/posts/quarkus-native-app
+
+Quarkus is advertised as a cloud-native runtime for Microservices but the link:https://quarkus.io/blog/quarkus-1-4-final-released/[introduction of command-mode] might have opened up new and intresting spaces where it could find adoption.
```

Comment:
```suggestion
Quarkus is advertised as a cloud-native runtime for Microservices but the link:https://quarkus.io/blog/quarkus-1-4-final-released/[introduction of command-mode] might have opened up new and interesting spaces where it could find adoption.
```